docs/tips.rst

```diff
@@ -246,7 +246,52 @@ Here's the complete list of workflow inputs:
 |                  |          | your GitHub Pages site, which is correct   |        |               |
 |                  |          | in most cases.                             |        |               |
 +------------------+----------+--------------------------------------------+--------+---------------+
+| ``deploy``       | No       | This is used to determine whether you will | bool   | ``true``      |
+|                  |          | deploy the site or not to GitHub Pages.    |        |               |
+|                  |          | This is most useful if you want to test a  |        |               |
+|                  |          | change to your website in a pull request   |        |               |
+|                  |          | before deploying those change.             |        |               |
++------------------+----------+--------------------------------------------+--------+---------------+
+
+Testing your build in a GitHub Pull Request
+"""""""""""""""""""""""""""""""""""""""""""
+
+If you want to test your build in a pull request before deploying to GitHub, your workflow might look something like this:
+
+.. code-block:: yaml
 
+    name: Build and Deploy Site
+    on:
+      push:
+        branches: ["main"]
+      pull_request:
+        branches: ["main"]
+      workflow_dispatch:
+        inputs:
+          deploy:
+            required: false
+            default: true
+            description: "Whether to deploy the site. If checked then build the site and deploy it. If not checked then just test that  the site builds successfully but don't deploy anything."
```

Comment:
`If checked then […]` 👉 `If checked, then […]`

`If not checked then […]` 👉 `If not checked, then […]`

`just test that  the` 👈  there is an extra space between "that" and "the"